controllers/user.js

```diff
@@ -39,6 +39,34 @@ async function add(req, res) {
   });
 }
 
+function getShareByEmail(req) {
+
+  const subject = encodeURIComponent(req.fluentFormat("share-by-email-subject"));
+  const body = encodeURIComponent(req.fluentFormat("share-by-email-message"));
+
+  return {
+    "gmail" : {
+      client: "Gmail",
+      class: "gmail",
+      href: `https://mail.google.com/mail/?view=cm&fs=1&su=${subject}&body=${body}`,
+    },
+    "yahoo" : {
+      client: "Yahoo",
+      class: "yahoo",
+      href: `https://compose.mail.yahoo.com/?subject=${subject}&body=${body}`,
+    },
+    "outlook" : {
+      client: "Outlook",
+      class: "outlook",
+      href: `https://outlook.live.com/owa/?path=/mail/action/compose&to=service%40domain.com&subject=${subject}&body=${body}`,
```

Comment:
Should drop the `to=service%40domain.com` url param so the compose message isn't pre-filled with this in the `To:` field.